kinetic/distribution.yaml

```diff
@@ -3273,6 +3273,22 @@ repositories:
       url: https://github.com/UTNuclearRoboticsPublic/move_base_to_manip.git
       version: kinetic
     status: maintained
+  move_basic:
+    doc:
+      type: git
+      url: https://github.com/UbiquityRobotics/move_basic.git
+      version: kinetic-devel
+    release:
+      tags:
+        release: release/kinetic/{package}/{version}
+      url: https://github.com/UbiquityRobotics-release/move_basic-release.git
+      version: 0.1.0-0
+    source:
+      test_pull_requests: true
```

Comment:
Have you followed the instructions referenced by bloom to provide access to the @ros-pull-request-builder user for this?